interfaces/Portalicious/src/app/domains/payment/payment.api.service.ts

```diff
@@ -23,7 +23,10 @@ export class PaymentApiService extends DomainApiService {
     });
   }
 
-  getPayment(projectId: Signal<number>, paymentId: Signal<number | undefined>) {
+  getPayment(
+    projectId: Signal<number | undefined>,
+    paymentId: Signal<number | undefined>,
+  ) {
     return this.generateQueryOptions<PaymentAggregate>({
       path: [...BASE_ENDPOINT(projectId), paymentId],
       enabled: () => !!paymentId(),
```

Comment:
With regards to my previous comment:

```suggestion
      path: [...BASE_ENDPOINT(projectId as Signal<number>), paymentId],
      enabled: () => !!projectID() && !!paymentId(),
```